entity-types/infra-awsprivatelinkendpoint/summary_metrics.yml

```diff
@@ -0,0 +1,50 @@
+providerAccountName:
+  title: AWS account
+  unit: STRING
+  tag:
+    key: providerAccountName
+region:
+  title: Region
+  unit: STRING
+  tag:
+    key: aws.awsRegion
+activeConnections:
+  goldenMetric: activeConnections
+  title: Active Connections
+  unit: COUNT
+packetsDropped:
+  goldenMetric: packetsDropped
+  title: Packets Dropped
+  unit: COUNT
+bytesProcessed:
+  goldenMetric: bytesProcessed
+  title: Bytes Processed
+  unit: BYTES
+newConnections:
+  goldenMetric: newConnections
+  title: New Connections
+  unit: COUNT
+rstPacketsReceived:
+  goldenMetric: rstPacketsReceived
+  title: Rst Packets Received
+  unit: COUNT
+activeConnections:
+  goldenMetric: activeConnections
+  title: Active Connections
+  unit: COUNT
+packetsDropped:
+  goldenMetric: packetsDropped
+  title: Packets Dropped
+  unit: COUNT
+rstPacketsReceived:
+  goldenMetric: rstPacketsReceived
+  title: Rst Packets Received
+  unit: COUNT
+bytesProcessed:
```

Comment:
Here we have repeated summary metrics

Reply:
Yeah, sorry, I was fixing the script that generates this files